### PR TITLE
UI of the B2B sample application is updated

### PR DESCRIPTION
**Pull request by Achintha444**
Screenshot 2022-11-08 at 15 07 46
Screenshot 2022-11-08 at 15 07 39

### CODE REVIEW

**Review by thivi — APPROVED** [review]

File: b2b-sample/guardio-insurance-sample-application/components/common/logo/logo.js [hunk 1]
```diff
@@ -22,31 +22,49 @@ import logoImage from "../../../public/logo.png";
...
             case "x-large":
-                return { width: "600px" };
+                imageStyle["width"] = "600px" ;
+
+                break;
             default:
                 break;
         }
-    }
+
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/components/common/logo/logo.js:57]: Let's remove redundant new lines. Check other places too.

File: b2b-sample/guardio-insurance-sample-application/pages/api/dashboard/me.js [hunk 2]
```diff
@@ -16,29 +16,37 @@
  * under the License.
  */
 
-import config from "../../../config.json";
 import getDataHeader from "../../../util/util/apiUtil/getDataHeader";
 import { dataNotRecievedError, notPostError } from "../../../util/util/apiUtil/localResErrors";
+import { getOrgUrl } from "../../../util/util/orgUtil/orgUtil";
 
+/**
+ * backend API call to get logged user details
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/pages/api/dashboard/me.js:24]: `Backend API call to get logged user details.` formatting issues. Check other places too.

Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/pages/api/dashboard/me.js:24]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/8b059f39358c5652ea99e473f4cf34e23ad95e0b

File: b2b-sample/guardio-insurance-sample-application/components/common/logo/logo.js (hunk 1 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/components/common/logo/logo.js:57]: Fixed with https://github.com/wso2/samples-is/pull/416/commits/acb244b9a8bb35ce524de853271162c3ffa83c4d

**Pull request merged**